From 576d03da0e33cdc31535cd724daaf1cdde3ac902 Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Wed, 8 Jul 2020 16:33:38 +0200
Subject: [PATCH] Read radeon-profile-daemon-server socket from
 /run/radeon-profile-daemon/

Gentoo has moved the radeon-profile-daemon-server socket file into that
subdir in order to provide secure access permissions.

Signed-off-by: Lars Wendler <polynomial-c@gentoo.org>
---
 radeon-profile/daemonComm.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/radeon-profile/daemonComm.cpp b/radeon-profile/daemonComm.cpp
index 21ec38f..dff00a0 100644
--- a/radeon-profile/daemonComm.cpp
+++ b/radeon-profile/daemonComm.cpp
@@ -34,7 +34,7 @@ void DaemonComm::sendConnectionConfirmation() {
 void DaemonComm::connectToDaemon() {
     qDebug() << "Connecting to daemon...";
     signalSender->abort();
-    signalSender->connectToServer("/run/radeon-profile-daemon-server");
+    signalSender->connectToServer("/run/radeon-profile-daemon/radeon-profile-daemon-server");
 }
 
 void DaemonComm::disconnectDaemon() {
-- 
2.27.0

